ReplayKitExample/ReplayKitExample/PresentationViewController.swift

```diff
@@ -0,0 +1,641 @@
+        camera.startCapture(device: device, format: cameraFormat) { (device, format, error) in
+
+        }
+
+        self.cameraSource = camera
+        self.localVideoTrack = videoTrack
+    }
+
+    func publishCamera() {
+        guard let participant = self.room?.localParticipant else {
+            return
+        }
+        participant.delegate = self
+
+        guard let videoTrack = self.localVideoTrack else {
+            return
+        }
+
+        let publishOptions = LocalTrackPublicationOptions(priority: .low)
+        participant.publishVideoTrack(videoTrack, publicationOptions: publishOptions)
+    }
+
+    func connectToPresentation() {
+        TwilioVideoSDK.setLogLevel(.info)
+
+        UIApplication.shared.isIdleTimerDisabled = true
+
+        let videoOptions = VideoBandwidthProfileOptions { (builder) in
+            // Minimum subscribe priority of Dominant Speaker's RemoteVideoTracks
+            builder.dominantSpeakerPriority = .standard
+
+            // Maximum bandwidth (Kbps) to be allocated to subscribed RemoteVideoTracks
+            builder.maxSubscriptionBitrate = 6000
+
+            // Max number of visible RemoteVideoTracks. Other RemoteVideoTracks will be switched off
+            builder.maxTracks = 4
+
+            // Subscription mode: collaboration, grid, presentation
+            builder.mode = .presentation
+
+            // Configure remote track's render dimensions per track priority
+            let renderDimensions = VideoRenderDimensions()
+
+            // Desired render dimensions of RemoteVideoTracks with priority low.
+            renderDimensions.low = VideoDimensions(width: 640, height: 480)
+
+            // Desired render dimensions of RemoteVideoTracks with priority standard.
+            renderDimensions.standard = VideoDimensions(width: 640, height: 480)
+
+            // Desired render dimensions of RemoteVideoTracks with priority high.
+            renderDimensions.high = VideoDimensions(width: 1920, height: 1080)
+
+            builder.renderDimensions = renderDimensions
+
+            // Track Switch Off mode: .detected, .predicted, .disabled
+            builder.trackSwitchOffMode = .predicted
+        }
+        let profile = BandwidthProfileOptions(videoOptions: videoOptions)
+        let connectOptions = ConnectOptions(token: accessToken!) { builder in
+            builder.bandwidthProfileOptions = profile
+
+            if let audioTrack = LocalAudioTrack() {
+                builder.audioTracks = [audioTrack]
+                self.localAudioTrack = audioTrack
+            }
+
+            // Use the preferred signaling region
+            if let signalingRegion = Settings.shared.signalingRegion {
+                builder.region = signalingRegion
+            }
+
+            // Viewers will publish smaller "thumbnail" videos at lower bandwidth
+            builder.encodingParameters = EncodingParameters(audioBitrate: 16, videoBitrate: 400)
+        }
```

Comment:
Need to use simulcast otherwise presentation mode will switch off all tracks except for the high priority screenshare. I have noticed that once the screenshare starts these video tracks are quickly switched off.